conferences/abstracts/LIBER-submission.md

```diff
@@ -0,0 +1,25 @@
+# Is this the way to reproducibility?
+
+## Submission Details
+Call for papers/Conference website: https://liberconference.eu/liber-2019-call-for-papers/  
+ID: 189  
```

Comment:
Move this section to the end to conform to other abstracts under heading `Submission Info`